main.nf

```diff
@@ -62,24 +62,23 @@ process kraken {
 		set id, species, platform, file(fastq_r1), file(fastq_r2) from fastq_kraken
 
 	output:
-		set id, species, platform, file("${id}.kraken") into kraken_export
+		set id, species, platform, file("${id}.bracken") into kraken_export
 
 	script:
 		read_params = fastq_r2.name == 'SINGLE_END' ?
 			"$fastq_r1" :
 			"--paired $fastq_r1 $fastq_r2"
 
 	"""
-	kraken --fastq-input --check-names --preload \\
-		--gzip-compressed \\
-		--db ${params.krakendb} \\
-		--threads ${task.cpus} \\
-		--output kraken.out \\
-		$read_params
-
-	kraken-report --db ${params.krakendb} kraken.out > kraken.rep
-
-	est_abundance.py -k ${params.brackendb} -i kraken.rep -o ${id}.kraken
+    kraken2 \\
+			--gzip-compressed \\
+            --db ${params.krakendb} \\
+            --threads ${task.cpus} \\
+            --output ${id}.kraken.out \\
+            --report ${id}.kraken.report \\
+            $read_params
+
+    bracken -d /fs1/jonas/krakendb/krakenstd -r 150 -i ${id}.kraken.report -o ${id}.bracken
```

Comment:
Should this be ${params.krakendb} rather than hardcoded?

Reply:
Jup, I had it changed, but accidentally not commited the last version of the main.nf script

Reply:
You can commit and push to this branch and the pull request will update.

Reply:
Nvm, I changed it.